examples/configuration/UsingSSHToConfigureNameResolution.ipynb

```diff
@@ -214,11 +277,27 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Wait for the nodes to become active!\n",
+    "**Wait for the nodes to become active!** That means running"
```

Comment:
https://github.com/ChameleonCloud/jupyterhub-deploy/blob/master/singleuser/bashrc.d/chameleon_functions.sh#L113

There is also https://github.com/ChameleonCloud/jupyterhub-deploy/blob/master/singleuser/bashrc.d/chameleon_functions.sh#L93 for SSH connectivity, once a floating IP is assigned.

Reply:
Let's do it in a future version; we ran out of time here.